autogpt/config/config.py

```diff
@@ -83,11 +83,6 @@ class Config(SystemSettings):
     plugins: list[str]
     authorise_key: str
 
-    # Executed immediately after init by Pydantic
-    def model_post_init(self, **kwargs) -> None:
```

Comment:
No longer needed as we initialize the `PluginsConfig` with the rest of the `Config` arguments